test/e2e/network/dns.go

```diff
@@ -170,7 +170,7 @@ var _ = SIGDescribe("DNS", func() {
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
-	It("should resolve DNS of partial qualified names for services ", func() {
+	It("should resolve DNS of partial qualified names for services [LinuxOnly]", func() {
```

Comment:
ditto.

Reply:
Done. Included as a comment.

Reply:
So, one of the names below - headlessService.Name - should resolve OK on windows, shouldn't it? As I recall:

name [ works in windows and linux ]
name.namespace [ linux only ]
name.namespace.svc [ linux only ]
_http._tcp... [ linux only ]

If we don't also split this out then, we will be missing a test case on windows, won't we?

Reply:
The ``_http._tcp..`` thing is not LinuxOnly, That is being tested by the test: ``should provide DNS for services``, and it's passing on Windows.

See: https://testgrid.k8s.io/sig-windows#aks-engine-azure-1-14-windows

Reply:
But I assume that is an FQDN in that test? It would be linux only for PQDN. My point wasn't that one, but rather that "name" should work in both, because it has no dots. If we mark this whole test as LinuxOnly, I think we lose coverage of that in Windows. Or is it hit by some other test?